Content.Client/CharacterInfo/Components/CharacterInfoComponent.cs

```diff
@@ -98,11 +98,6 @@ public CharacterInfoControl(IResourceCache resourceCache)
                     }
                 });
 
-                AddChild(new Placeholder()
-                {
-                    PlaceholderText = Loc.GetString("character-info-health-and-status-effects-text")
```

Comment:
should also remove the corresponding loc string since its not being used

Reply:
do you mean this?
https://github.com/space-wizards/space-station-14/pull/5206/files#diff-5df9c78eca9ac3854f25e2d01a74ac1bce2aad8ec1653d1084234fa0093393bdL106-L110

Reply:
nah i mean since you removed the only GetString call for `character-info-health-and-status-effects-text` in https://github.com/space-wizards/space-station-14/blob/master/Resources/Locale/en-US/character-info/components/character-info-component.ftl

Reply:
oh worm gotcha

Reply:
done, should be there